jest.config.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
       //isolatedModules: true
     }],
   },
-  testRegex: ['/test/.*-test.*.ts$'],
+  testRegex: ['/test/.*-test.*.ts$', '/test/.*\.test\.ts$', '/test/.*\.spec\.ts$'],
```

Comment:
Instead of changing this, can we just rename the test files in sparqlee to be aligned?